src/schema/v2/schema.ts

```diff
@@ -203,6 +203,7 @@ import { DeletePageMutation } from "./Page/DeletePageMutation"
 import { UpdatePageMutation } from "./Page/UpdatePageMutation"
 import { createArtistMutation } from "./artist/createArtistMutation"
 import { CollectorProfilesConnection } from "./CollectorProfile/collectorProfiles"
+import { verifyAddressMutation } from "./verify_address"
```

Comment:
Should this actually be a mutation? If nothing on the server is changing it might be better as a query.

It can still use a POST for the data loader (POST may be appropriate for REST b/c it is sending data to the server for processing, irrespective of modifying or creating data), but I think mutations in GraphQL are a little bit more explicit about modifying server-side data.

Reply:
makes sense, thanks for the suggestion Matt! my thinking was around being consistent with what the BE exposes, but there's nothing modified on the db 👍